application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -70,13 +70,37 @@
       ; &lt;label for="XPoll.XPollClass_0_name"&gt;$doc.displayPrettyName('name', false, false)&lt;/label&gt;
       : #displayPollTitle()
     #end
-    ; &lt;label for="XPoll.XPollClass_0_description"&gt;$doc.displayPrettyName('description', false, false)&lt;/label&gt;
-    : $doc.display('description')
-    ; &lt;label for="XPoll.XPollClass_0_status"&gt;$doc.displayPrettyName('status', false, false)&lt;/label&gt;
-    : $doc.display('status')
-    ; &lt;label for="XPoll.XPollClass_0_proposals"&gt;$doc.displayPrettyName('proposals', false, false)&lt;/label&gt;
-      (% class="xHint" %)$services.localization.render('XPoll.XPollClass_proposals_hint')
-    : $doc.display('proposals')
+    (% class="row" %)
+    (((
+    (% class="col-xs-12 col-sm-6" %)
+      (((
+      ; &lt;label #if($xcontext.action == 'edit')for="XPoll.XPollClass_0_proposals"#end&gt;
+          $doc.displayPrettyName('proposals', false, false)
+        &lt;/label&gt;
+        (% class="xHint" %)$services.localization.render('XPoll.XPollClass_proposals_hint')
+      : $doc.display('proposals')
+      )))
+      (% class="col-xs-12 col-sm-6" %)
+      (((
+      ; &lt;label #if($xcontext.action == 'edit')for="XPoll.XPollClass_0_status"#end&gt;
+          $doc.displayPrettyName('status', false, false)
+        &lt;/label&gt;
+        (% class="xHint" %)$services.localization.render('XPoll.XPollClass_status_hint')
+      #if($xcontext.action == 'view')
+        : &lt;span class="label label-warning"&gt;$doc.display('status')&lt;/span&gt;
```

Comment:
Should the color change depending on the status?

Reply:
Right now the status is displayed only when the poll is "in preparation". So when it is active or finished, there is no "Status" displayed.